client/src/constants/actionTypes.js

```diff
@@ -73,3 +73,7 @@ export const APP_IMAGE_SET_LOGO_SUCCESS = 'APP_IMAGE_SET_LOGO_SUCCESS'
 export const APP_IMAGE_SET_LOGO_ERROR = 'APP_IMAGE_SET_LOGO_ERROR'
 
 export const SNACKBAR_EMPTY = 'SNACKBAR_EMPTY'
+
+export const CHANNELS_LOADED = 'CHANNELS_LOADED'
+export const CHANNELS_LOAD = 'CHANNELS_LOAD'
+export const CHANNELS_ERROR = 'CHANNELS_ERROR'
```

Comment:
I know the app is not very consistent with action names, but I like the `SUCCESS` and `ERROR` postfix. We also use those prefixes for optimistic actions, so it would be nice follow - even though this couldn't be optimistic.

Reply:
@Birkbjo so CHANNELS_LOAD_SUCCESS instead of CHANNELS_LOADED for example, right? I agree in that case, will change it